litmus-portal/frontend/src/views/ChaosWorkflows/BrowseSchedule/TableData.tsx

```diff
@@ -246,6 +219,88 @@ const TableData: React.FC<TableDataProps> = ({ data, deleteRow }) => {
           </div>
         </Popover>
       </TableCell>
+      <TableCell>
+        <Button
+          onClick={handlePopOverClickForSchedule}
+          style={{ textTransform: 'none' }}
+        >
+          <span
+            className={
+              YAML.parse(data.workflow_manifest).spec.suspend === true
+                ? classes.dark
+                : ''
+            }
+          >
+            <div className={classes.expDiv}>
+              <Typography className={classes.expInfo}>
+                <strong>Show Schedule</strong>
+              </Typography>
+              {isOpenSchedule ? (
+                <KeyboardArrowDownIcon />
+              ) : (
+                <ChevronRightIcon />
+              )}
+            </div>
+          </span>
+        </Button>
+        <Popover
+          id={idSchedule}
+          open={isOpenSchedule}
+          anchorEl={popAnchorElSchedule}
+          onClose={handlePopOverCloseForSchedule}
+          anchorOrigin={{
+            vertical: 'bottom',
+            horizontal: 'center',
+          }}
+          transformOrigin={{
+            vertical: 'top',
+            horizontal: 'center',
+          }}
+          style={{
+            marginTop: 10,
+          }}
+        >
+          <div className={classes.weightDiv}>
+            <Typography className={classes.scheduleDetailsFlex}>
+              <strong>Starting Date:</strong>
+              <span className={classes.scheduleDetailsValue}>
+                {formatDate(data.created_at)}
+              </span>
+            </Typography>
+            <Typography className={classes.scheduleDetailsFlex}>
+              <strong>Last Run:</strong>
+              <span className={classes.scheduleDetailsValue}>
+                {timeDifferenceForDate(data.updated_at)}
+              </span>
+            </Typography>
+            <Typography className={classes.scheduleDetailsFlex}>
+              <strong>Regularity:</strong>
+              <span className={classes.scheduleDetailsValue}>
+                {data.cronSyntax === '' ? (
+                  <>{t('chaosWorkflows.browseSchedule.regularityOnce')}</>
+                ) : (
+                  cronstrue.toString(data.cronSyntax)
+                )}
+              </span>
+            </Typography>
+          </div>
+        </Popover>
+      </TableCell>
+      <TableCell>
+        <span
+          className={
+            YAML.parse(data.workflow_manifest).spec.suspend === true
```

Comment:
=== true might not be needed.

Reply:
Suspend is a string, ESlint is giving errors if === not used.